bobtemplates/plone/indexer.py

```diff
@@ -37,7 +37,7 @@ def _update_indexers_configure_zcml(configurator):
     match_xpath = "zope:include[@package='.{0}']".format(configurator.variables['indexer_name'])
     match_str = '-*- extra stuff goes here -*-'
     insert_str = """
-  <include package=".{0}" />
+  <include file="{0}.zcml" />
 """.format(configurator.variables['indexer_name'])
```

Comment:
Shouldn't this just be `<include package=".indexers" />`?

Reply:
No, what you say is done some lines before, here: https://github.com/plone/bobtemplates.plone/blob/0e925fa682ae59decd5807e3250fb387713ff248/bobtemplates/plone/indexer.py#L19

Reply:
Ok, nevermind then :)
I just have no clue of what is happening here!
Please ignore me :)

Reply:
This exact method modifies the `configure.zcml` file created inside the `indexers` folder and adds a reference to the newly created indexer. But it adds as a "include package" instead of "include file", that's all.